api/jobs.go

```diff
@@ -15,6 +19,7 @@ var NewID = func() string {
 }
 
 var serverErrorMessage = "internal server error"
+var ErrJobNotFound = "the job id could not be found in the jobs collection"
```

Comment:
This should belong to the job store package - sorry I should have been clearer in my previous comment.

Reply:
ok, done